crates/recursion/circuit/src/machine/compress.rs

```diff
@@ -508,7 +510,7 @@ impl<SC: BabyBearFriConfig> SP1CompressWitnessValues<SC> {
 }
 
 impl SP1CompressWitnessValues<BabyBearPoseidon2> {
-    pub fn dummy<A: MachineAir<BabyBear>>(
+    pub fn dummy<A: MachineAir<BabyBear> + Air<SymbolicAirBuilder<BabyBear>>>(
```

Comment:
How come this trait bound is needed?

The other additional `Air<SymbolicAirBuilder>` ones make sense since they already have `Air<AirBuilder>` bound, but this originally didn't have a AirBuilder related bound.